### ISSUE TITLE
Steam opens non-functioning windows when account has an account alert

### ISSUE THREAD

**Issue by LeLobster** [ValveSoftware/steam-for-linux#7029]
#### Your system information

* Steam client version (build number or date): Apr 8 2020
* Distribution (e.g. Ubuntu): Debian
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: No, but is irrelevant

#### Please describe your issue in as much detail as possible:
Today I signed in to the Steam client while having an Account Alert active (the yellow one about key revocation) and this caused Steam to display 3 extra windows which were non-functioning.  
They were each about ~100px big and only a black color was displayed.  
The names of the windows were: Friends, Servers & Webchat  
When trying to close them it would completely close out the entire Steam client instance in stead of just the window.  

I've confirmed this is actually caused by the Account Alert since I didn't click the "I have read this message" checkbox and restarted Steam, which caused the 3 window to appear again.  
but once I clicked the checkbox and restarted Steam the 3 windows didn't show up anymore.  

#### Steps for reproducing this issue:

1. Have a key revocation alert on your account
2. Start Steam

**Comment by h1stoire** [comment]
Hello, I can reproduce this bug the same way OP did just by logging in.
I was wondering if there was a fix to it but visibly there isn't. 
From what I've heard so far it has something to do with web containers from the Friends UI and such. There is also a similar issue still opened there: #5584 

I'm not sure of anything but I would love it if it gets finally fixed before, for my case I will not close this account alert window, for the little story I've been wrongly VAC banned and after fighting my way out of it I want to keep this window as a souvenir that "VAC" like any other anti-cheat isn't 100% accurate.

I have a fresh installation of Solus, I wanted to move from Windows and go back to Linux and try Solus in that case. 
Being my first time using GitHub to post an issue I am unfortunately not sure what kind of informations I am supposed to give so please feel free to ask more if needed.

What I can tell so far is how to reproduce this bug.
As OP (@LeLobster) said you just have to login to your account with an Account alert
It pops up as usual but as soon as it does 3 windows will open up aswell, they will get stuck on the top-left corner of the screen with no way to close them.
They're named "Friends","Servers" and "Web Chat Container"

_Names of the "ghost" windows_
![Screenshot from 2021-03-08 19-56-26](https://user-images.githubusercontent.com/21106215/110367765-670f7900-8048-11eb-9118-e8c100f8b1a7.png)

If I close any of them the Steam launcher will shutdown, even if the main Steam window is opened.

_Screenshot of my whole screen showing the "ghost" windows on the top left, the account alert and that Issue opened_
![image](https://user-images.githubusercontent.com/21106215/110366477-9fae5300-8046-11eb-8584-355bd267cf5e.png)

_About Steam Window_
![image](https://user-images.githubusercontent.com/21106215/110368328-30862e00-8049-11eb-8a9e-2a96d7cccaa3.png)

_Pastebin containing my System Informations from Steam_
https://pastebin.com/mrsDdbaw

_neofetch_
![image](https://user-images.githubusercontent.com/21106215/110368832-d89bf700-8049-11eb-8e18-ed0b717a21a7.png)

**Comment by PowerOfVoid** [comment]
I got this bug by unsubscribing from an MMO.

I had to restart Steam after doing what the alert wanted before the bug would go away.

**Comment by CrimsonFork** [comment]
Same issue with the notification for the Steam Deck.
![image](https://user-images.githubusercontent.com/22666602/162806353-a77d6c7e-4b1c-4c53-afce-67df8b4d332a.png)
![image](https://user-images.githubusercontent.com/22666602/162806685-e624de95-599c-4539-934a-81a77c6f81cc.png)
![image](https://user-images.githubusercontent.com/22666602/162806975-2f3b87fe-5ddb-409f-a668-01253c3bec89.png)
↑here the window was alt-dragged to the corner and then released
![image](https://user-images.githubusercontent.com/22666602/162809101-df5d0c2e-6074-48bd-815d-701d66bdc5fb.png)

**Comment by CrimsonFork** [comment]
Terminal output when the windows are opened:
```
Installing breakpad exception handler for appid(steam)/version(1647446817)
BuildCompleteAppOverviewChange: 271

(steam:22459): Gtk-WARNING **: 09:08:57.015: gtk_disable_setlocale() must be called before gtk_init()

(steam:22459): GLib-CRITICAL **: 09:08:57.015: unquote_string_inplace: assertion 'err == NULL || *err == NULL' failed
RegisterForAppOverview 1: 25ms
RegisterForAppOverview 2: 25ms
```
Terminal output when one of them is closed, making Steam quit completely:
```
(steam:22459): GLib-GObject-CRITICAL **: 09:09:50.808: g_object_unref: assertion 'G_IS_OBJECT (object)' failed

(steam:22459): GLib-GObject-WARNING **: 09:09:50.810: invalid (NULL) pointer instance

(steam:22459): GLib-GObject-CRITICAL **: 09:09:50.810: g_signal_handlers_disconnect_matched: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

(steam:22459): GLib-GObject-CRITICAL **: 09:09:50.811: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
[2022-04-12 09:09:50] Shutdown
```

**Comment by Xaekai** [comment]
Yep, definitely having this issue. Steam Deck alert causing this on Arch/KDE
![image](https://user-images.githubusercontent.com/594255/184447780-2ae7d7fa-9f45-4779-83f4-c6d22077e938.png)
